index.html

```diff
@@ -1144,13 +1172,29 @@ <h3>Errors</h3>
    both <a>editable</a> and <a>resettable</a>.
  </tr>
 
+ <tr>
+  <td><dfn>invalid event id</dfn>
+  <td>400
+  <td><code>invalid event id</code>
+  <td>Argument was an invalid event ID.
+ </tr>
+
  <tr>
   <td><dfn>invalid selector</dfn>
   <td>400
   <td><code>invalid selector</code>
   <td>Argument was an invalid selector.
  </tr>
 
+ <tr>
+  <td><dfn>invalid session creation job id</dfn>
+  <td>404
+  <td><code>invalid session creation job id</code>
+  <td>Occurs if the given <a>session creation job id</a> is not in the list of <a>session creation jobs</a>,
+   or is in the list but has a <a data-lt="session creation status">status</a> of
```

Comment:
While I can see the reasoning for this, it might be useful to be able to get the `cancelled` or `failed` event for inspection.